worker/src/zimfarm_worker/task/worker.py

```diff
@@ -424,7 +451,7 @@ def start_uploader(self, upload_dir: Path, filename: str):
         if not self.task or not self.host_task_workdir:
             logger.error("No task or host task workdir to start uploader")
```

Comment:
`to start uploader` => `to start ZIM uploader` (and fix method name to `start_zim_uploader`, remove the stupid `Gathering ZIM metadata ...` log (WTF?), and fix other log to mention `ZIM uploader`